lib/pushmi_pullyu/cli.rb

```diff
@@ -104,6 +104,10 @@ def parse_options(argv)
         opts[:piddir] = piddir
       end
 
+      o.on('-W', '--workdir PATH', 'Path where downloads, etc. are done ') do |workdir|
+        opts[:workdir] = workdir
```

Comment:
Yeah they probably not needed. They can just get folded into the default hash honestly